### PR TITLE
Add nanotube as an option on docker-compose

### PR DESCRIPTION
**Pull request by laq**
Include nanotube as a relay option. 
crelay is still available as a commented out option.

See https://github.com/bookingcom/carbonapi/issues/233

For this to work properly the nanotube import paths have to be fixed. https://github.com/bookingcom/nanotube/pull/9

### CODE REVIEW

**Comment by grzkv** [comment]
I think it makes sense to clean up `crelay`. Also, did you run some tests to verify if everything works?

File: docker/nanotube/nanotube.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cluster a34000
```
Comment (grzkv) [inline docker/nanotube/nanotube.conf]: I am not sure I understand the purpose of this file. It's a crelay config but looks like nanotube one. Could you please explain?

File: docker/nanotube/config/config.toml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# The settings are listed in this file are with their default values.
```
Comment (grzkv) [inline docker/nanotube/config/config.toml]: I think these comments are applicable mostly within the `nanotube` repo.

File: docker/nanotube/nanotube.conf (hunk 1 above)
Comment (laq) [reply docker/nanotube/nanotube.conf]: It was a left over from coping crelay config. Removed.

File: docker/nanotube/config/config.toml (hunk 2 above)
Comment (laq) [reply docker/nanotube/config/config.toml]: Just removed them

**Pull request merged**